src/main/java/sawkiewicz/task1/ShoppingCart.java

```diff
@@ -0,0 +1,41 @@
+package sawkiewicz.task1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShoppingCart {
+    String id;
+    List <CartItem> purchase = new ArrayList<>();
+
+    public ShoppingCart() {
+    }
+
+    public ShoppingCart(String id) {
+        this.id = id;
+    }
+
+    public ShoppingCart(String id, List<CartItem> purchase) {
+        this.id = id;
+        this.purchase = purchase;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public int calculateTotalCost() {
+        int totalCost = 0;
+
+        for (CartItem cartItem: purchase) {
+
+            int quantity = cartItem.quantity;
```

Comment:
if needed, use getters `cartItem.getQuantity()`, fields should be private

Reply:
done